app/controllers/rhapsody_realities.js

```diff
@@ -18,11 +20,25 @@ class RhapsodyController {
      * @param req
      * @param res
      */
-    index(req, res) {
+    index(req, res, next) {
         let viewData = {
             menuActive: 'rhapsody'
         };
-        res.render('show_rhapsody', viewData);
+        // Get today's date for rhapsody fetching
+        let today = dateFormat(now, "isoDate");
+        //get date to with day for display in view
+        let date = dateFormat(now, "fullDate");
```

Comment:
quite fishy that we are doing this for the second time, we could have reused `today` and formatted differently when passing to the viewData, but will let this slide for now